src/bin/pg_autoctl/pgsql.c

```diff
@@ -2605,15 +2628,15 @@ pgsql_one_slot_has_reached_target_lsn(PGSQL *pgsql,
 	 */
 
 	/* *INDENT-OFF* */
-	char *sql =
-		"   select $1::pg_lsn <= flush_lsn, flush_lsn "
-		"     from pg_replication_slots slot"
-		"     join pg_stat_replication rep"
-		"       on rep.pid = slot.active_pid"
-		"   where (   slot_name ~ '" REPLICATION_SLOT_NAME_PATTERN "' "
-		"          or slot_name = '" REPLICATION_SLOT_NAME_DEFAULT "') "
-		"     and sync_state in ('sync', 'quorum') "
-		"order by flush_lsn desc limit 1";
+    char *sql =
+            "   select $1::pg_lsn <= flush_lsn, flush_lsn "
+            "     from pg_replication_slots slot"
+            "     join pg_stat_replication rep"
+            "       on rep.pid = slot.active_pid"
+            "   where (   slot_name ~ '" REPLICATION_SLOT_NAME_PATTERN "' "
+            "          or slot_name = '" REPLICATION_SLOT_NAME_DEFAULT "') "
+            "     and sync_state in ('sync', 'quorum') "
+            "order by flush_lsn desc limit 1";
```

Comment:
This looks like a whitespace change that we should not have in this PR, please fix.